pinot-core/src/main/java/com/linkedin/pinot/core/query/executor/ServerQueryExecutorV1Impl.java

```diff
@@ -86,10 +87,11 @@ public void init(Configuration queryExecutorConfig, DataManager dataManager, Ser
   }
 
   @Override
-  public DataTable processQuery(final InstanceRequest instanceRequest) {
+  public DataTable processQuery(final QueryRequestContext queryRequestContext) {
```

Comment:
Can we have a QueryRequest class (which has the QueryContext, or may be just rename QueryContext to QueryRequest), so that this api seems more natural: processQuery takes a QueryRequest?